src/webui/src/components/NavCon.tsx

```diff
@@ -178,8 +179,8 @@ class NavCon extends React.Component<NavProps, NavState> {
                     </Stack>
                 </StackItem>
                 {/* the drawer for dispatcher & nnimanager log message */}
-                {isvisibleLogDrawer && <LogDrawer closeDrawer={this.closeLogDrawer} />}
-                <ExperimentDrawer isVisble={isvisibleExperimentDrawer} closeExpDrawer={this.closeExpDrawer} />
+                {isvisibleLogDrawer && <LogPanel closeDrawer={this.closeLogDrawer} />}
+                {isvisibleExperimentDrawer && <ExperimentPanel closeExpDrawer={this.closeExpDrawer} experimentProfile={EXPERIMENT.profile} />}
```

Comment:
I don't know if it's proper to pass `EXPERIMENT.profile` as property.
Because it's a getter function, the `ExperimentPanel` is likely to re-render every time.

Reply:
had tested this status. ExperimentPanel will refresh as interval not every time.